bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/ui/TabDebugger.java

```diff
@@ -1465,7 +1587,7 @@ public void setDefaults(ILaunchConfigurationWorkingCopy configuration) {
 			configuration.setAttribute(ConfigurationAttributes.GDB_SERVER_LOG,
 					DefaultPreferences.GDB_SERVER_LOG_DEFAULT);
 
-			defaultString = fPersistentPreferences.getGdbServerOtherOptions();
+			defaultString = null;
```

Comment:
Hi @kolipakakondal, fixed in the last commit. We don't need to set the gdb server other options here at all, so I removed it. Also, removed `fPersistentPreferences.putGdbServerOtherOptions(stringValue);` since we don't need the same other options between configurations